Makefile

```diff
@@ -341,7 +341,9 @@ credentials/pull-secret.json:
 
 .PHONY: lifecycle-agent-deploy
 lifecycle-agent-deploy: lifecycle-agent
-	KUBECONFIG=../$(SNO_KUBECONFIG) make -C lifecycle-agent install deploy
+	KUBECONFIG=../$(SNO_KUBECONFIG) \
+	IMG=$(LCA_IMAGE) \
```

Comment:
If we set IMG to an empty string will we get the default?

Reply:
nope, but there is a default LCA_IMAGE variable in the Makefile pointing to `quay.io/openshift-kni/lifecycle-agent-operator:latest`

Is latest a proper tag to default to?

Reply:
I guess LCA will handle it once 4.14.0 and latest diverge